src/app/submission/Database.ts

```diff
@@ -20,9 +20,9 @@ const submissionTableItemSchema = z.object({
   pk: dynamoDBStringSchema,
   sk: dynamoDBStringSchema,
   pdfKey: dynamoDBStringSchema,
-  dateSubmitted: dynamoDBStringSchema,
-  formName: dynamoDBStringSchema,
-  formTitle: dynamoDBStringSchema,
+  dateSubmitted: dynamoDBStringSchema.optional(),
+  formName: dynamoDBStringSchema.optional(),
+  formTitle: dynamoDBStringSchema.optional(),
   attachments: z.object({
     L: z.array(dynamoDBStringSchema),
```

Comment:
Gezien je SubmissionDataInterface die attachments als optional heeft, gaat deze dan wel goed als hij altijd een waarde verwacht?

Reply:
Dit is wat dubieus, het gaat goed, maar ik vermoed vooral omdat er nu altijd minimaal een lege array wordt weggeschreven. Ik maak deze ook optioneel, dan is het in ieder geval consistent.